ontology/yaml/resources/HVAC/entity_types/MAU.yaml

```diff
@@ -802,6 +802,17 @@ MAU_NONCANONICAL_4:
   - supply_fan_run_status
   - supply_fan_speed_percentage_command
 
+MAU_DFSS2X_DFVSC2X_CHWZC_HWZC_VOADM:
+  description: "Multi-zone MAU with chilled water and heating water control and outside air monitoring."
```

Comment:
The description says multi-zone but you are using discharge abstracts

Reply:
Single